enterprise_gateway/services/sessions/kernelsessionmanager.py

```diff
@@ -216,12 +214,26 @@ def load_sessions(self):
         raise NotImplementedError("KernelSessionManager.load_sessions() requires an implementation!")
 
     # abstractmethod
-    def save_sessions(self):
+    def load_session(self, kernel_id):
         """
-        Saves the sessions dictionary to persistent store.  Caller is responsible for synchronizing call.
+        Load and initialize _sessions member from persistent storage for a single kernel.  This method is called from
+        refresh_sessions().
+        """
+        raise NotImplementedError("KernelSessionManager.load_sessions() requires an implementation!")
+
+    # abstractmethod
+    def delete_sessions(self):
+        """
+        Persists the changes to the persistent storage.  Caller is responsible for synchronizing call.
```

Comment:
Fix the description to be relative to deletion.  Add a kernel_ids list parameter - depending on previous comment discussion.

Please update the `NotImplementedError` description below as well.  Thanks.

Reply:
Done.